lib/active_merchant/billing/gateways/paysafe.rb

```diff
@@ -21,6 +21,7 @@ def purchase(money, payment, options = {})
         post = {}
         add_auth_purchase_params(post, money, payment, options)
         add_airline_travel_details(post, options)
+        add_lodging_details(post, options)
```

Comment:
Looking at the Paysafe docs as we worked through some of the other changes, we determined that this (and also the airline method) should probably be moved to the `add_auth_purchase_params` method, since the fields appear to be available on both auth and purchase - the primary difference between auth/purchase on this gateway is the `settleWithAuth` tag that gets added below